test/test_api.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+import os
+import mock
+
+from behave_testrail_reporter.api import APIClient
+
+
+class APIClientTestCase(unittest.TestCase):
+    def test_env_variables_not_present(self):
+        with self.assertRaises(ValueError) as context:
+            APIClient(base_url='index.php/')
+
+        self.assertTrue(
+            'Environment variables to authenticate on Testrail API are not defined!' in str(context.exception)
```

Comment:
This could be 
```python
expected = 'Environment variables to authenticate on Testrail API are not defined!'
with self.assertRaisesMessage(ValueError, expected):
    APIClient(base_url='index.php/')
```

Reply:
Oops -- I didn't realise that `assertRaisesMessage` was from django :sheep:

Reply:
This could use `self.assertIn` though.